### PR TITLE
Add Ansible Operator testing guide

### PR DESCRIPTION
**Pull request by fabianvf**
**Description of the change:**

Adds documentation for ansible e2e testing. Followup to #861 

**Motivation for the change:**

### CODE REVIEW

**Comment by AlexNPavel** [comment]
The links aren't formatted correctly. They need to start with `https://` otherwise markdown thinks it's a local/relative path link (which is why the tests are failing)

**Comment by fabianvf** [comment]
as pointed out in #941, you may need python2-devel and python-libselinux depending on the distro you are using. Need to add that to the requirements section

File: doc/ansible/dev/testing_guide.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+# Testing Ansible Operators with Molecule
+
+## Getting started
+
+### Requirements
+To begin, you sould have:
```
Comment (dymurray) [inline doc/ansible/dev/testing_guide.md]: I think we should also include a note about the version of docker needed. According to https://github.com/moby/moby/pull/31352 it is in v17.05

**Pull request merged**